app/templates/base.html

```diff
@@ -68,4 +78,45 @@
         </style>
     </body>
 
+    <script src="https://code.jquery.com/jquery-3.5.1.js"#}
+                integrity="sha256-QWo7LDvxbWT2tbbQ97B53yJnYU3WhH/C8ycbRAkjPDc="
+                crossorigin="anonymous">
+    </script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js">
+    </script>
+
+    <script>
+{#        {% for language in AVAILABLE_LANGUAGES.items() %}#}
+{#             {% if CURRENT_LANGUAGE == language[0] %}#}
+{#                 {{ language[1] }}#}
+{#             {% else %}#}
+{#                 <a href="{{ url_for('set_language', language=language[0]) }}" >{{ language[1] }}</a>#}
+{#             {%  endif %}#}
+{#        {% endfor %}#}
+
+        function site_lang(lang){
+            var data = [{'language': lang}]
+
+            $.ajax({
+              type: "POST",
+              url: "/process_lang",
+              data: JSON.stringify(data),
+              contentType: "application/json",
+              dataType: 'json',
+              success: function(result) {
+                  console.log("THING" + result);
+                  setTimeout(function(){
```

Comment:
I now realize we might not actually need this bit, I'll do some testing and remove it if unneeded. Having it won't break it and should only be a miniscule delay